frontend/src/pages/Home/style.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const HomeContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  row-gap: 45px;
+  background-color: #15171c;
+  margin: 0 30px;
+  box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.32);
+  padding: 60px;
+  min-width: 500px;
+  max-width: 1300px;
+  color: #e1e9fc;
+`;
+
+export const Header = styled.h2`
+  font-size: 70px;
+`;
+
+export const HomeContent = styled.div`
+  display: flex;
+  column-gap: 45px;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const CacheImage = styled.img`
```

Comment:
The current style makes the image not-so-well cropped
![Screenshot from 2022-03-13 22-11-49](https://user-images.githubusercontent.com/23612331/158079460-150c1cbe-28f2-4855-999d-90fccdd8d780.png)